src/modules/75-cf/strings/strings.en.yaml

```
@@ -186,6 +186,8 @@ featureFlags:
     onRequest: On Request
     addTargeting: Add Targeting
     specificTargeting: Specific Targeting
+    ruleChangeModalTitle: Changing default rule?
+    ruleChangeModalDescription: 'Changing the default rule will result in the value of {{ variationValue }} being served to all SDK clients unless there are specific rules in place. <br/><br/> Change the state anyway?'
```

would it be worth making the `variationValue` bold? just for clarity bc I found it was a lot of text in the modal
```suggestion
    ruleChangeModalDescription: 'Changing the default rule will result in the value of <strong>{{ variationValue }}</strong> being served to all SDK clients unless there are specific rules in place. <br/><br/> Change the state anyway?'
```

---

good ppoint, thanks have updated